zkevm-circuits/src/evm_circuit/execution/precompiles/modexp.rs

```diff
@@ -1032,7 +1195,7 @@ mod test {
                         PUSH32(word!("0x0000000000000000000000000000000000000000000000000000000000000008"))
                         PUSH1(0x60)
                         MSTORE
-                        PUSH32(word!("0x0000000000000000000000000000000000000000000000000000000000000009"))
+                        PUSH32(word!("0x1000000000000000000000000000000000000000000000000000000000000009"))
```

Comment:
this line is used to test modexp gas for large input?

Reply:
There were no test cases where `exponent`'s bit length was significant enough for the gas cost to be > 200. So this line changes the exponent's bit length to a high value, hence testing the gas calculation gadget.